crates/node/src/vmm/qemu.rs

```diff
@@ -244,7 +244,7 @@ impl Provider for Qemu {
             .args(["-qmp", &format!("tcp:localhost:{qmp_port},server")]);
 
         // TODO: When GPU argument handling is refactored, this should be fixed as well.
```

Comment:
The todo is still revelant?

Reply:
Yeah. It's a bit terse but the gist of that is basically that there's currently no tracking of _which_ GPU is allocated and which are available. This hack below shall work because the machines will only have one GPU for now, but in the future this needs improving once we get there.